src/Microsoft.DotNet.Tools.New/Program.cs

```diff
@@ -24,13 +30,27 @@ private static string GetFileNameFromResourceName(string s)
             return parts[parts.Length - 2] + "." + parts[parts.Length - 1];
         }
 
-        public int CreateEmptyProject()
+        public int CreateEmptyProject(Language language)
```

Comment:
The enum conversion does seem a little weird, but directly using input as a directory name seems dangerous to me. I can't think of a situation where this causes a vulnerability, but directory traversal via user input has been known to cause scary things :smile: